### DISCUSSION TITLE
api_cli which no longer works

### DISCUSSION THREAD

**Discussion by gpointer** [drakkan/sftpgo#702] [General]
Hi ,
we want to go productive with sftpgo soon. I have to type in all the user information manually. I originally did that with the Cli_api, which no longer works. Is there another way to do a user update etc. at the system level?
Setting up swagger and openapi is beyond my skills, I'm not a developer.

Thanks and greetings Gina

**Comment by drakkan** [comment]
Hi,

swagger ui is already installed by default: `http://localhost:8080/openapi/swagger-ui/`. It also prints the curl commands to make API calls. I have no time to also maintain a CLI tool sorry. Maybe I could re-add this support in the future if someone want to sponsor its development and maintenance.

There are some third party tools:

https://github.com/ramnes/sftpgo-client
https://pypi.org/project/ant-sftpgo-client/1.0.1/

I haven't tested them

**Reply by gpointer** [reply]
Dear drakkan, thank you for quick reponse. ok  I type in manually, I'm only a little it-admin ;-)
